contracts/SpokePool.sol

```diff
@@ -44,7 +44,7 @@ abstract contract SpokePool is Testable, Lockable, MultiCaller {
     mapping(address => mapping(uint256 => bool)) public enabledDepositRoutes;
 
     struct RelayData {
-        address sender;
+        address depositor;
```

Comment:
Unrelated: naming this to `depositor` is more clear and better delineates between "sender" and "relayer"

Reply:
+1